src/octokit/get-octokit-throttle-options.ts

```diff
@@ -1,11 +1,11 @@
 import Bottleneck from "bottleneck";
-import Redis from "ioredis";
+import * as redis from "ioredis";
```

Comment:
Why not use named exports? Avoid mixing the "import * as" import style.

```suggestion
import { Redis, RedisOptions } from "ioredis";
```